modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -94,3 +94,24 @@ Using `salt-thin` can be enabled as a fallback method, but it requires Python 3
 This method is not recommended nor supported and exists for development purposes only.
 Set [option]``web.ssh_use_salt_thin`` to `true` in the [path]``/etc/rhn/rhn.conf`` configuration file.
 ====
+
+== Extending Salt Bundle functionality with Python packages using pip
+
+The Salt Bundle includes [literal]``pip`` to make it possible to extend the functionality of the bundled Salt Minion with extra Python packages.
```

Comment:
```suggestion
The Salt Bundle includes [literal]``pip`` to make it possible to add features to the Salt Bundle connection method with extra Python packages.
```
Your version can be read as if this is a possibility to add Python packages to the Client in general.

By all means, we must avoid the term "minion".  We prefer the term "client" here.

Last I'm wondering whether we should say "Python packages" or "Python modules" (pip has probably something to do with packages).

Reply:
No, now it's not correct. Salt Bundle is not a connection method by itself, it's more like a different flavour of Salt Minion.

Python package here is not the RPM or DEB package of the linux, but the package of Python (a set of modules). And we are talking here only about extending the bundled python with extra packages which will be available for the bundled python and salt bundle (bundled salt minion to be exact).

```suggestion
The Salt Bundle includes [literal]``pip`` to make it possible to extend the functionality of the Salt Bundle with extra Python packages.
```
But here I would like to avoid to mention Salt Bundle twice in the same sentance, so in original I've replaced the second occurance with the bundled Salt Minion as it's actually the same in this context.

Reply:
thanks, I think I now understand it.  We can close this as resolved.